### PR TITLE
Adrien mode selector

### PR DESCRIPTION
**Pull request by acarrou**

### CODE REVIEW

File: subsystem-drive/source/main.cpp [hunk 1]
```diff
@@ -15,14 +19,18 @@ drive_commands SerialEnterCommands()
...
+    CommandLerper lerp;
     while (1)
     {
         commands = SerialEnterCommands();
         commands.Print();
         sjsu::Delay(2s);
-
-        tri_wheel_router_arguments arguments = steermode.DriveSteering(commands);
+        
+        arguments = tri_wheel.SetLegArguments(ModeSelect::SelectMode(lerp.Lerp(mode_switch.SwitchSteerMode(commands, arguments))));
```
Comment (naterpotatoers) [inline subsystem-drive/source/main.cpp:33]: Should make lerp class static like we did for steer modes

**Review by naterpotatoers — APPROVED** [review]
We will need to write some behavioral tests for this soon

File: subsystem-drive/source/main.cpp (hunk 1 above)
Comment (Coreyboy1820) [reply subsystem-drive/source/main.cpp:33]: We can't, the lerp class needs to hold previous information which i'm pretty sure ruins the point of the static.

**Comment by Coreyboy1820** [comment]
I'll make some tests for this soon. Just wanted to get this code out there and fixed.

**Pull request merged**